src/components/InstanceFilters/InstanceFilters.js

```diff
@@ -1,39 +1,59 @@
-import React from 'react';
+import React, { useEffect, useState, useRef } from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage, useIntl } from 'react-intl';
+import _ from 'lodash';
 
-import { useStripes } from '@folio/stripes/core';
 import {
   Accordion,
+  AccordionSet,
   FilterAccordionHeader,
   languageOptions,
 } from '@folio/stripes/components';
 import {
   DateRangeFilter,
 } from '@folio/stripes/smart-components';
+import { languageOptionsES } from '@folio/stripes-components/util/languages';
 
 import {
   retrieveDatesFromDateRangeFilterString,
   makeDateRangeFilterString,
 } from '../../utils';
 import { DATE_FORMAT } from '../../constants';
+import {
+  DISCOVERY_SUPPRESS,
+  EFFECTIVE_LOCATION,
+  INSTANCE_FORMAT_ID,
+  INSTANCE_TAGS,
+  INSTANCE_TYPE_ID,
+  LANGUAGES,
+  MODE_OF_ISSUANCE_ID,
+  NATURE_OF_CONTENT_TERM_IDS,
+  SOURCE,
+  STAFF_SUPPRESS,
+  LANGUAGE,
+  RESOURCE,
+  FORMAT,
+  MODE,
+  NATURE_OF_CONTENT,
+  TAGS,
+} from './constants';
 import TagsFilter from '../TagsFilter';
 import CheckboxFacet from '../CheckboxFacet';
 
 const InstanceFilters = props => {
   const {
     activeFilters: {
-      effectiveLocation = [],
-      resource = [],
-      language = [],
-      format = [],
-      mode = [],
-      natureOfContent = [],
-      discoverySuppress = [],
-      staffSuppress = [],
+      effectiveLocation,
```

Comment:
And here you can apply the imported constants

Reply:
Thanks, made changes.